src/components/Match/Container.tsx

```diff
@@ -26,11 +26,13 @@ const Container: React.FC = () => {
   };
   return (
     <>
+    <ThemeProvider theme={theme}>
```

Comment:
You don`t need to use     <ThemeProvider theme={theme}> twice. In the components, you can use const theme = useTheme() hook to access theme